code-challenges.test.js

```diff
@@ -10,43 +10,164 @@
+    return value.toLowerCase().includes(letter);
+  });
+};
+
+// Test Suites: 1 passed, 1 total
+// Tests:       2 passed, 2 total
+
 // --------------------3) Create a function that takes in an array of 5 numbers and determines whether or not the array is a "full house". A full house is exactly one pair and one three of a kind.
 
 // a) Create a test with expect statements using the variable provided.
 
-const hand1 = [5, 5, 5, 3, 3]
-// Expected output: true
-const hand2 = [5, 5, 3, 3, 4]
-// Expected output: false
-const hand3 = [5, 5, 5, 5, 4]
-// Expected output: false
-const hand4 = [7, 2, 7, 2, 7]
-// Expected output: true
+describe("ohIThoughtYouMeantTheTvShow", () => {
+  const hand1 = [5, 5, 3, 5, 3];
+  // Expected output: true
+  const hand2 = [5, 5, 3, 3, 4];
+  // Expected output: false
+  const hand3 = [5, 5, 5, 5, 4];
+  // Expected output: false
+  const hand4 = [7, 2, 7, 2, 7];
+  // Expected output: true
+  const hand5 = [2, 2, 2, 3, 3];
+  it("takes in an array of 5 numbers and determines whether or not the array is a 'full house'", () => {
+    expect(ohIThoughtYouMeantTheTvShow(hand1)).toEqual(true);
+    expect(ohIThoughtYouMeantTheTvShow(hand2)).toEqual(false);
+    expect(ohIThoughtYouMeantTheTvShow(hand3)).toEqual(false);
+    expect(ohIThoughtYouMeantTheTvShow(hand4)).toEqual(true);
+    expect(ohIThoughtYouMeantTheTvShow(hand5)).toEqual(true);
+  });
+});
+
+// ReferenceError: ohIThoughtYouMeantTheTvShow is not defined
 
 // b) Create the function that makes the test pass.
+
+// pseudo:
+// create a function called ohIThoughtYouMeantTheTvShow that takes in an array of 5 numbers
+// within our function we can use .sort((a,b) => a - b) on our argument array (you've seen me explain this in every assessment for the past three weeks, for brevity i will refrain from doing it again lol)
+// now that our array is sorted from least to greatest, we want to define two variables in our function, these two variables will essentially define what both variations of a 'full house' are.
+// the first variable will be called firstTwoAndLastThreeMatch which will be defined as array[0] === array[1] && array[2] === array[4]. This is saying that the FIRST TWO indexes of our sorted array hold the same value, AND the LAST THREE indexes hold the same value.
+// the second variable will be called firstThreeAndLastTwoMatch which will be defined as array[0] === array[2] && array[3] === array[4]. This variable is saying that the FIRST THREE indexes hold the same value, AND the LAST TWO indexes hold the same value.
+// finally we want to return firstTwoAndLastThreeMatch || firstThreeAndLastTwoMatch which will return true if our argument array matches either of those varables.
+// this is an incredibly specific/static solution to the prompt, but given the specific nature of the prompt (the array will have 5 numbers), I am comfortable with this solution. :)
+
+const ohIThoughtYouMeantTheTvShow = (array) => {
+  array.sort((a, b) => a - b);
```

Comment:
Awesome! I love this approach, it's a bit different than a conditional statement but basically with the same output.